Assets/_BuildingBlocks/FloatingOrigin/Scripts/WorldTransform.cs

```diff
@@ -8,15 +8,10 @@ public class WorldTransform : MonoBehaviour, IHasCoordinate
     {
         [SerializeField] private WorldTransformShifter worldTransformShifter;
         [SerializeField] private CoordinateSystem referenceCoordinateSystem = CoordinateSystem.RDNAP;
-        public Coordinate Coordinate {
-            get;
-            set;
-        }
-        public Quaternion Rotation
-        {
-            get;
-            set;
-        }
+
+        public Coordinate Coordinate { get; set; } = new(CoordinateSystem.RDNAP);
```

Comment:
Ik meen me te herinneren dat deze constructor kapot was. Kan je even met Martijn dubbelchecken of dat opgelost is? Ik heb hier een keer uren aan besteed omdat ik niet begreep wat er fout was.

Reply:
Oh, ik heb geen problemen ervaren bij testen, maar goed dat je dat dan aangeeft. @mvangog: weet jij dit toevallig nog?